containers/src/Kokkos_OffsetView.hpp

```diff
@@ -1182,9 +1219,11 @@ class OffsetView : public ViewTraits<DataType, Properties...> {
       const Kokkos::Impl::ViewCtorProp<P...>& arg_prop,
       std::enable_if_t<!Kokkos::Impl::ViewCtorProp<P...>::has_pointer,
                        typename traits::array_layout> const& arg_layout,
-      const index_list_type minIndices)
+      const index_list_type minIndices, bool strided_from_padding   = false)
       : m_track(),
-        m_map()
+        m_map(),
+         m_is_strided_from_padding(Kokkos::Impl::ViewCtorProp<P...>::allow_padding ||
+                                  strided_from_padding)
```

Comment:
... or is there any problem if arg_prop allows padding in case strided_from_padding is passed?

Reply:
That is a good point, ViewCtorProp is a better location for the member.